plugin/metrics/prometheus/metricsstore/reader_test.go

```diff
@@ -217,17 +215,17 @@ func TestGetLatencies(t *testing.T) {
 			wantDescription: "0.95th quantile latency, grouped by service & operation",
 			wantLabels: map[string]string{
 				"service_name": "emailservice",
+				"operation":    "/OrderResult",
 			},
 			wantPromQlQuery: `histogram_quantile(0.95, sum(rate(span_metrics_duration_bucket{service_name =~ "emailservice", ` +
-				`span_kind =~ "SPAN_KIND_SERVER"}[10m])) by (service_name,span_name,le))`,
+				`span_kind =~ "SPAN_KIND_SERVER"}[10m])) by (service_name,operation,le))`,
```

Comment:
We need to keep `span_name`. 

From: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/spanmetricsconnector/README.md#span-to-metrics-processor-to-span-to-metrics-connector

> The operation metric attribute was renamed to span.name.

I think we need to update this to `span_name` (and rename the file for consistency): https://github.com/jaegertracing/jaeger/blob/c0fb781662eacb4ec6f544e78d081528541c2473/plugin/metrics/prometheus/metricsstore/testdata/service_operation_datapoint_response.json#L8

So that we're mocking the metrics data response from the spanmetrics connector, which uses `span_name` instead of `operation`.

Reply:
Okay! will make changes